### PR TITLE
Feature: watch stylesheet if passed with --style arg and reload on stylesheet change

### PR DESCRIPTION
**Pull request by gabeboning**
Did it primarily for my purposes; hence I didn't bother to go so far as to watch stylesheet if it's included in the YAML front matter, but certainly makes my life a lot easier!

### CODE REVIEW

File: bin/cleaver [hunk 1]
```diff
@@ -66,13 +66,23 @@ program
...
+      createAndSave([file], options);
     });
+    
+    // also watch on stylesheet if included via command prompt
+    if('style' in options) {
+      fs.watchFile(options.style, { persistent: true,  interval: 100 }, function () {
+        console.log('Rebuilding: ' + new Date());
+        createAndSave([file], options);
+      });
+
```
Comment (jdan) [inline bin/cleaver]: Mind removing the empty line here?

File: bin/cleaver [hunk 2]
```diff
@@ -66,13 +66,23 @@ program
...
+    
     console.log('Watching for changes on ' + file + '. Ctrl-C to abort.');
     fs.watchFile(file, { persistent: true, interval: 100 }, function () {
       console.log('Rebuilding: ' + new Date());
-      createAndSave([file], getOptions());
+      createAndSave([file], options);
     });
+    
+    // also watch on stylesheet if included via command prompt
+    if('style' in options) {
```
Comment (jdan) [inline bin/cleaver]: Space after `if` please :)

**Comment by jdan** [comment]
Works for me :) We should think about doing a first-pass of sorts that would allow us to get the front-matter styles for free.

Two small nits and then I'll happily merge! Thanks for this.

**Comment by gabeboning** [comment]
Also added a log that stylesheet was being watched. Don't see any reason not to. 

Anywhere, there ya go! Cheers.

**Comment by jdan** [comment]
Woot!

**Pull request merged**